framework/include/variables/MooseVariableDataFV.h

```diff
@@ -417,6 +431,8 @@ class MooseVariableDataFV : public MooseVariableDataBase<OutputType>, public Mes
   using MooseVariableDataBase<OutputType>::_nodal_value_dot_old;
   using MooseVariableDataBase<OutputType>::_nodal_value_dotdot_old;
   using MooseVariableDataBase<OutputType>::_required_vector_tags;
+
+  friend void Moose::initDofIndices<>(MooseVariableDataFV<OutputType> &, const Elem &);
```

Comment:
Can this be inlined? Also, whenever I see friend functions in moose it is really restrictive (like `Controls` and `ControllableParameters` is this going to be similarly restrictive?

Reply:
`initDofIndices` is a template function defined slightly higher up in the diff. Since it is templated it is implicitly marked as `inline`. What do you mean by similarly restrictive? The template function has been declared as a friend of this class so it can access private/protected data members.